org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/groovy/SmartApp.groovy

```diff
@@ -160,17 +171,20 @@ def registerDeviceChange() {
         if(subscriptionEndpt != null){
 			if(state.deviceSubscriptionMap[deviceId] == null){
 				state.deviceSubscriptionMap.put(deviceId, [subscriptionEndpt])
-				log.info "Added subscription URL: ${subscriptionEndpt} for ${myDevice.displayName}"
```

Comment:
can you keep all the log statements that you have deleted? unless they are incorrect/wrong